target_chains/stylus/contracts/pyth-receiver/src/integration_tests.rs

```diff
@@ -0,0 +1,372 @@
+#[cfg(test)]
+mod test {
+    use crate::error::PythReceiverError;
+    use crate::test_data;
+    use crate::PythReceiver;
+    use alloy_primitives::{address, Address, I32, I64, U256, U64};
+    use motsu::prelude::*;
+    use pythnet_sdk::wire::v1::{AccumulatorUpdateData, Proof};
+    use wormhole_contract::WormholeContract;
+    const TEST_PRICE_ID: [u8; 32] = [
+        0xe6, 0x2d, 0xf6, 0xc8, 0xb4, 0xa8, 0x5f, 0xe1, 0xa6, 0x7d, 0xb4, 0x4d, 0xc1, 0x2d, 0xe5,
+        0xdb, 0x33, 0x0f, 0x7a, 0xc6, 0x6b, 0x72, 0xdc, 0x65, 0x8a, 0xfe, 0xdf, 0x0f, 0x4a, 0x41,
+        0x5b, 0x43,
+    ];
+    const TEST_PUBLISH_TIME: u64 = 1751563000;
+    const TEST_PRICE: i64 = 10967241867779;
+    const TEST_CONF: u64 = 4971244966;
+    const TEST_EXPO: i32 = -8;
+    const TEST_EMA_PRICE: i64 = 10942391100000;
+    const TEST_EMA_CONF: u64 = 4398561400;
```

Comment:
it'd be nice if you move these to test_data too (like a function returns update data and what actually is inside of it?)